yarn.lock

```diff
@@ -6220,6 +6228,11 @@ flat-cache@^3.0.4:
     flatted "^3.1.0"
     rimraf "^3.0.2"
 
+flatpickr@^4.0.6:
```

Comment:
@mahmoudtahadev
![Screenshot 2022-03-16 at 10 04 52](https://user-images.githubusercontent.com/13088968/158554818-cd91e27e-e4b0-4064-902d-79093af9db71.png)
make sure we only have 1 flatpicker version